src/App.tsx

```diff
@@ -12,159 +19,26 @@ export const App = () => (
     >
       <div className="container">
         <div className="navbar-brand">
-          <a className="navbar-item" href="#/">
-            Home
-          </a>
-
-          <a
-            className="navbar-item has-background-grey-lighter"
-            href="#/people"
-          >
-            People
-          </a>
+          <PageNavLink to="/" text="Home" />
+          <PageNavLink to="/people" text="People" />
         </div>
```

Comment:
4. Add the `Navbar` with 2 links `Home` and `People`. Active one should have the `has-background-grey-lighter` class;

Create a separate component for navigation links